app/celer-g4/GlobalSetup.cc

```diff
@@ -186,12 +186,14 @@ void GlobalSetup::ReadInput(std::string const& filename)
 #endif
     }
 
+#if CELERITAS_USE_JSON
```

Comment:
The code here will still compile if JSON is missing. Is it failing a validation somewhere? Just change the line below to:
```c++
if (CELERITAS_USE_JSON  && input_.output_file.empty())
```

Reply:
That was actually a "mistake" that I forgot to test. I saw that we needed a build option condition for this bit while reading the code, added the macro, and something got me sidetracked before testing turning off the json build option.